control/src/lib.rs

```diff
@@ -286,62 +319,37 @@ pub mod pallet {
 			access: AccessModel,
 			fee_model: FeeModel,
 			fee: T::Balance,
-			gov_asset: u8,
-			pay_asset: u8,
+			gov_asset: T::CurrencyId,
+			pay_asset: T::CurrencyId,
 			member_limit: u64,
 			deposit: T::Balance,
```

Comment:
What about making this parameter optional and use `T::InitialDeposit::get()` in case it's not provided?

Reply:
maybe also a `T::MinimumDeposit` could be more accurate